src/ui/src/app/Service/employee-service.service.ts

```diff
@@ -0,0 +1,26 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable, inject } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class EmployeeServiceService {
+
+  constructor() { }
+  private http = inject(HttpClient);
+  employeeId = 1;
+
+
+  getHierarchy() {
+    return this.http.get("https://localhost:7015/api/Employee/hierarchy");
+  }
+  getEmployeeProfile() {
+    return this.http.get("https://localhost:7015/api/Employee/by-Id?EmployeeId=1");
```

Comment:
https://localhost:7015/api/Employee, it is common for all the apis can we make it as class variable like baseUrl and reuse it here